build-parent/pom.xml

```diff
@@ -192,6 +193,18 @@
                 <version>${commonmark.version}</version>
             </dependency>
 
+            <!-- Extension processor -->
+            <dependency>
+                <groupId>org.jboss.jdeparser</groupId>
+                <artifactId>jdeparser</artifactId>
+                <version>${jdeparser.version}</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.javaparser</groupId>
+                <artifactId>javaparser-core</artifactId>
+                <version>${javaparser-core.version}</version>
+            </dependency>
```

Comment:
I was actually thinking about doing it the other way around (define both in `quarkus-extension-processor`), assuming that no other module is going to need either of the two.
Having them in `build-parent` affects 900 modules (on a version bump) and "only" 655 when in processor.

But I guess this way it's more in line with other existing dependencies.

Reply:
Yeah, agreed, but I favored consistency.